### DISCUSSION TITLE
Should ToggleButton has a pending state.

### DISCUSSION THREAD

**Discussion by phuoccss** [adobe/react-spectrum#7210] [General]
A pending state has been recently introduced in RAC Button. Does it make sense to have the same for ToggleButton?

**Comment by reidbarber** [comment]
Seems like a reasonable feature request. Feel free to create an issue, or convert this discussion into one.

**Reply by phuoccss** [reply]
Thanks for looking into this. Created https://github.com/adobe/react-spectrum/issues/7256
